### PR TITLE
Installable as pod + other minor fixes

### PR DESCRIPTION
**Pull request by johannth**
This PR introduces the following changes to the Flurry Adapter for MoPub
- Makes the adapter installable as a [Cocoapod](https://cocoapods.org/) so developers don't have to copy and modify the source code on installation. In order to make this a nicer experience, we replaced the embedded Flurry API key within the `FlurryMPConfig` singleton with `+(void)initializeWithFlurryAPIKey:(NSString *)flurryAPIKey` which developers will need to call on app load. We also updated the readme to explain the changes. Note that this is a backwards incompatible change so we should probably bump a version to the next major version after merging.
- Removes some dead code from `FlurryNativeCustomEvent`. (900d33a19a0991054950595e30aa7921beb23fa9)
- Passes along non-standard native ad properties to MoPub so that client code can check if there are keys with `flurry_` prefix and use the additional data to render richer native ads. (155d90046020a53a215b65aaaa1f65d94eac1206)
- Makes sure that `FlurryInterstitialCustomEvent` calls `[self.delegate interstitialCustomEventDidAppear]` in `adInterstitialDidRender` according to [MoPub documentation](https://github.com/mopub/mopub-ios-sdk/blob/master/MoPubSDK/MPInterstitialCustomEventDelegate.h#L105). (fc1a9e95cd83404c02d622f3f7239dd7af8f51fb)
- Tries its best to normalize the star rating to a range that MoPub expects. I had a slightly hard time finding good documentation on what you could expect to receive from `apprating` because the ads that we got returned `@"90"` but the [documentation](https://developer.yahoo.com/flurry/docs/publisher/code/ios/) hints at `@"90/100"`. Anyways, MoPub expects `@(4.5)` and we try our best to only pass along star ratings of the correct type and range. (9b97a7fe11c1e40f2f4252ca0e1ec101d11919c7)

Please let me know if all (or at least some) of these changes would be mergeable upstream. We're willing to spend time to get it into a mergeable state so that we don't have to maintain our own fork.

### CODE REVIEW

File: FlurryAdapterForMoPubiOS.podspec [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Pod::Spec.new do |s|
+  s.name             = "FlurryAdapterForMoPubiOS"
+  s.version          = "7.2.1"
+  s.summary          = "Flurry iOS Adapter for MoPub"
+  s.homepage         = ""
+  s.license          = ""
```
Comment (johannth) [inline FlurryAdapterForMoPubiOS.podspec:6]: This is something that someone at Flurry needs to fill out if you decide to go forward with these changes.

**Comment by enginoid** [comment]
I've verified that the following scenarios work correctly:
+ Real-time game: MoPub native at index 0
+ Real-time game: AdMob native at index 0 (does not display ad, as expected)
+ Real-time game: Flurry native at index 0
+ Real-time game: MoPub interstitial on close
+ Real-time game: AdMob interstitial on close
+ Real-time game: Flurry interstitial on close
+ Async game: MoPub interstitial on close
+ Real-time game: Native on close
+ Async game: Flurry native at index 0

**Comment by ugonna** [comment]
Thanks for the PR. We currently have no plans of maintaining this adapter in CocoaPods, although that may change in the future.

In the meantime, could you break down this PR and reset the CocoaPods related changes (or just rebase your current head and remove b4d4640)?

**Comment by johannth** [comment]
@Ugonna Thanks for the review. I'm going to then close this PR in favour of #2. Could you have another look at that one?

**Pull request closed without merging**